modules/distribution/resources/api_templates/velocity-template_synapse-config_versioned-webapp.xml

```diff
@@ -0,0 +1,270 @@
+<!--
```

Comment:
Please note that this diff of the file 'velocity-template_synapse-config_versioned-webapp.xml'  is the result of moving, renaming, & code-formatting of the old file '[modules/distribution/resources/api_templates/velocity_template.xml](https://github.com/wso2/product-app-manager/pull/234/files#diff-20e21fea2ad1a1eb1057ee3a8d12d676L1)'.